test_gpios.py

```diff
@@ -20,9 +20,26 @@ async def asyncSetUp(self):
         self.robot = await RobotClient.at_address(conf.address, opts)
         board = Board.from_robot(self.robot, "board")
         self.input_pin = await board.gpio_pin_by_name(conf.INPUT_PIN)
-        self.interrupt = await board.digital_interrupt_by_name(conf.INPUT_PIN)
         self.output_pin = await board.gpio_pin_by_name(conf.OUTPUT_PIN)
 
+        # Most boards have combination GPIO/PWM/interrupt pins. However, rarely
+        # they are separated to different pins (e.g., the Beaglebone AI-64 does
+        # not have GPIO functionality on the PWM pins or vice versa). If you
+        # define them separately, define these extra variables in the config.
+        # Otherwise, we'll reuse the same inputs and outputs.
+        try:
+            INTERRUPT_PIN = conf.INTERRUPT_PIN
+        except AttributeError:
+            INTERRUPT_PIN = conf.INPUT_PIN
+
+        try:
+            PWM_PIN = conf.PWM_PIN
+        except AttributeError:
+            PWM_PIN = conf.OUTPUT_PIN
```

Comment:
This way, we don't lose the testing functionality for pins function switching. We can switch on all the other boards while still being able to test on the beaglebone. Nice.